server/controllers/finance/reports/generalLedger/index.js

```diff
@@ -66,22 +66,23 @@ function renderAccountSlip(req, res, next) {
 
   let report;
 
-  try {
-    report = new ReportManager(ACCOUNT_SLIP_TEMPLATE, req.session, options);
-  } catch (e) {
-    return next(e);
-  }
-
   const data = {};
 
-  return AccountReport.getAccountTransactions(params.account_id, GENERAL_LEDGER_SOURCE)
+  return Accounts.lookupAccount(params.account_id)
+    .then((account) => {
+      _.extend(data, { account });
+
+      options.filename = 'Extrait'.concat('_', account.number, '_', account.label);
```

Comment:
The `ReportManager` add automatically a date at the end of the filename.

![csv_export_name](https://cloud.githubusercontent.com/assets/5445251/25476344/c0969b1a-2aee-11e7-8bff-b6275d412237.png)